README.md

```diff
@@ -1,39 +1,317 @@
-# Open Source Project Template
+# Review Service
 
-This repository contains a template to seed a repository for an Open Source
-project.
+This repository introduces abstractions around native review capabilities to ease code sharing and testability.
+It also introduces business logic to quickly configure conditions and state tracking to prompt for reviews at the right moment.
 
-## How to use this template
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE) ![Version](https://img.shields.io/nuget/v/ReviewService.Abstractions?style=flat-square) ![Downloads](https://img.shields.io/nuget/dt/ReviewService.Abstractions?style=flat-square)
```

Comment:
I don't think you want to reference the nuget (.NET version) ones here

Reply:
Right I forgot the todos